themes/vscode/vscode.theme.sh

```diff
@@ -26,4 +26,4 @@ _omb_theme_PROMPT_COMMAND() {
     unset -f _omb_theme_PROMPT_COMMAND
 }
 _omb_theme_PROMPT_COMMAND
-export PROMPT_DIRTRIM=4
\ No newline at end of file
+export PROMPT_DIRTRIM=4
```

Comment:
We want to add a newline at the end of the file to make it better work with POSIX utilities.